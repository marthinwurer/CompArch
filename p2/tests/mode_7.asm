;
; tests addressing mode 7
;
; uses instructions MOV, HALT, BPT
;
; uses other addressing modes 0, 6
;
	.globl	start
. = . + 50
v1:	.word	10
v2:	.word	v1
. = . + 50
;
start:	mov	@v2,r1
	bpt
	halt
;
	.end	start
