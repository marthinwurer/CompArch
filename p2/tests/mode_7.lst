      1                                 ;
      2                                 ; tests addressing mode 7
      3                                 ;
      4                                 ; uses instructions MOV, HALT, BPT
      5                                 ;
      6                                 ; uses other addressing modes 0, 6
      7                                 ;
      8 000000                                   .globl  start
      9 000050                          . = . + 50
     10 000050  000010                  v1:      .word   10
     11 000052  000050'                 v2:      .word   v1
     12 000124                          . = . + 50
     13                                 ;
     14 000124  017701  177722          start:   mov     @v2,r1
     15 000130  000003                           bpt
     16 000132  000000                           halt
     17                                 ;
     18 000134  000124'                          .end    start


pc     = 000007 %  
r0     = 000000 %  
r1     = 000001 %  
r2     = 000002 %  
r3     = 000003 %  
r4     = 000004 %  
r5     = 000005 %  
r6     = 000006 %  
r7     = 000007 %  
sp     = 000006 %  
start    000124 RG 
v1       000050 R  
v2       000052 R  
