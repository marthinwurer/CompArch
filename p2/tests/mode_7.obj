028 2 08 00
02a 2 28 00
054 4 c1 1f d2 ff
058 2 03 00
05a 2 00 00
054
