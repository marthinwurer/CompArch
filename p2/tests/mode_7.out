CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 124
000124:  PS[00]  017701  MOV ????,R1

Machine Halted - mode error

Simulated time 8 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
